website/content/en/docs/add-ons/load-balancer/guide.md

```diff
@@ -90,6 +99,8 @@ If you choose DNS validation for the validation of the certificates, you will be
 
 Set up resources required for the Load Balancer controller:
 
+#### 1. Verify subnet configurations
```

Comment:
can we drop the numbering since other sections in this guide dont have it? afaik, in terraform this is also configured. they can skip entire configure LBC section

Reply:
done